src/bootstrap/src/core/config/config.rs

```diff
@@ -581,6 +581,7 @@ pub struct Target {
     pub wasi_root: Option<PathBuf>,
     pub qemu_rootfs: Option<PathBuf>,
     pub no_std: bool,
+    pub codegen_backends: Option<Vec<Interned<String>>>,
```

Comment:
Can you add a change info for this field? Similar to https://github.com/rust-lang/rust/blob/04521fd10e6eefb113ca1cefd02862599fd08640/src/bootstrap/src/utils/change_tracker.rs#L74-L78

Reply:
Done